modopt/tests/test_opt.py

```diff
@@ -407,14 +407,15 @@ def setUp(self):
         self.combo = proximity.ProximityCombo([self.identity, self.positivity])
         if import_sklearn:
             self.owl = proximity.OrderedWeightedL1Norm(weights.flatten())
-
         self.ridge = proximity.Ridge(linear.Identity(), weights)
         self.elasticnet_alpha_0 = proximity.ElasticNet(linear.Identity(),
                                                        alpha=0,
                                                        beta=weights)
         self.elasticnet_beta_0 = proximity.ElasticNet(linear.Identity(),
                                                       alpha=weights,
                                                       beta=0)
+        self.one_support = proximity.KSupportNorm(beta=0.2, k_value=1)
```

Comment:
I might need to change the beta parameter so it basically rescales wit the ridge in case of k_value = dimension

Reply:
Are you planning to make this change in the current PR?

Reply:
That was a general question

Reply:
I think you are the expert in this context so I have defer to your judgement. Either way I will be happy to merge the PR.

Reply:
I'm making it compatible with the ridge, it will ease the comparison between the two methods.